examples/simple.go

```diff
@@ -26,17 +26,20 @@ var simpleQs = []*survey.Question{
 }
 
 func main() {
-	answers := struct {
-		Name  string
-		Color string
-	}{}
+	//	answers := struct {
```

Comment:
Can we leave this file as is? I much prefer showing the struct pattern as the default one since it avoids unnecessary type casting and lends to cleaner code.